__test__/auth.test.js

```diff
@@ -0,0 +1,89 @@
+const request = require('supertest');
+
+const app = require('../index');
+
+const token =
+  'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJleHAiOjE1NjU5NzQ1ODUsImlhdCI6MTU2NTExMDU4NSwic3ViIjoiNWQ0OTU1MWRkNjQ4ODhjOTYzNmMwZjI1IiwiZW1haWwiOiJva2V0ZWdhaEBnbWFpbC5jb20ifQ.Fd5NhXaalDNhv_8wAMX3hsSI0tcApE-t6-wrU7cciTg';
```

Comment:
This should be created by the user 
```suggestion
 let user = new User({... data});
 let token = user.generateToken()
```
because at some point the user id may differ.